gui/object.cpp

```diff
@@ -41,12 +41,13 @@ GuiObject::~GuiObject() {
 void GuiObject::reflowLayout() {
 	if (!_name.empty()) {
 		int16 w, h;
-
-		if (!g_gui.xmlEval()->getWidgetData(_name, _x, _y, w, h) || w == -1 || h == -1) {
+		bool useRTL = true;
```

Comment:
Something is wrong with whitespace here.

Reply:
I couldn't find out the issue here. I deleted this and wrote it again, but git diff didn't show any change